spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -2002,6 +2003,11 @@ def show_code_analysis_results(self, line_number, block_data):
             msg = msg.strip()
             # Avoid messing TODO, FIXME
             msg = msg[0].upper() + msg[1:]
+            msg = textwrap.wrap(msg, width=60)

```

Comment:
CodeEditor inherits from TextEditBaseWidget, which in turn inherits from BaseEditMixin. Therefore you can use here `_DEFAULT_MAX_WIDTH` instead of a fixed width of 60.